src/components/NewCommentForm/NewCommentForm.jsx

```diff
@@ -1,39 +1,70 @@
-import React from 'react';
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { addComment } from '../../api/comments';
 import './NewCommentForm.scss';
 
-export const NewCommentForm = () => (
-  <form className="NewCommentForm">
-    <div className="form-field">
-      <input
-        type="text"
-        name="name"
-        placeholder="Your name"
-        className="NewCommentForm__input"
-      />
-    </div>
+export const NewCommentForm = ({ selectedPostId, newRenderOfComments }) => {
+  const [userName, setUserName] = useState('');
+  const [userEmail, setUserEmail] = useState('');
+  const [comment, setComment] = useState('');
 
-    <div className="form-field">
-      <input
-        type="text"
-        name="email"
-        placeholder="Your email"
-        className="NewCommentForm__input"
-      />
-    </div>
+  const addNewComment = async() => {
+    await addComment(selectedPostId, userName, userEmail, comment);
+    newRenderOfComments(selectedPostId);
+  };
 
-    <div className="form-field">
-      <textarea
-        name="body"
-        placeholder="Type comment here"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <button
-      type="submit"
-      className="NewCommentForm__submit-button button"
+  return (
+    <form
+      className="NewCommentForm"
+      onSubmit={(event) => {
+        event.preventDefault();
+        addNewComment();
+      }}
     >
-      Add a comment
-    </button>
-  </form>
-);
+      <div className="form-field">
+        <input
+          type="text"
+          name="name"
+          placeholder="Your name"
+          className="NewCommentForm__input"
+          value={userName}
+          onChange={({ target }) => setUserName(target.value)}
```

Comment:
```suggestion
          onChange={({ target }) => setCommentPart(target.name, target.value)}
```
create method setCommentPart which should work like this:
```javascript
const setCommentPart = (key, value) => {
  setComment(prevComment => ({
    ...prevComment,
    [key]: value,  
  })
}
```